### DISCUSSION TITLE
Does Rancher Desktop on MacOS uses or needs ~/Library/Containers/com.docker.docker/Data/vms/0/data/?

### DISCUSSION THREAD

**Discussion by TheJavaGuy** [rancher-sandbox/rancher-desktop#6640] [Q&A] [answered]
I use MacOS 14.3. I switched from Docker Desktop to Rancher Desktop and noticed that the directory `~/Library/Containers/com.docker.docker/Data/vms/0/data/` takes 64GB! I assume it's a leftover from Docker Desktop and that I could safely delete it, but I'm not sure. Does Rancher Desktop need this directory at all?

**Comment by mook-as** [comment, marked as answer]
Nope, we use `/Users/<you>/Library/Application Support/rancher-desktop/lima/0/` instead.
